src/utils/Permissions.ts

```diff
@@ -3,37 +3,42 @@ import { APIGuildMember, APIOverwrite, OverwriteType, Snowflake } from 'discord-
 import { CachedGuild, DiscordEventMap } from '../typings/Discord'
 
 export const bits = {
-  createInvites: 0x00000001,
-  kick: 0x00000002,
-  ban: 0x00000004,
-  administrator: 0x00000008,
-  manageChannels: 0x00000010,
-  manageGuild: 0x00000020,
-  addReactions: 0x00000040,
-  auditLog: 0x00000080,
-  prioritySpeaker: 0x00000100,
-  stream: 0x00000200,
-  viewChannel: 0x00000400,
-  sendMessages: 0x00000800,
-  tts: 0x00001000,
-  manageMessages: 0x00002000,
-  embed: 0x00004000,
-  files: 0x00008000,
-  readHistory: 0x00010000,
-  mentionEveryone: 0x00020000,
-  externalEmojis: 0x00040000,
-  viewInsights: 0x00080000,
-  connect: 0x00100000,
-  speak: 0x00200000,
-  mute: 0x00400000,
-  deafen: 0x00800000,
-  move: 0x01000000,
-  useVoiceActivity: 0x02000000,
-  nickname: 0x04000000,
-  manageNicknames: 0x08000000,
-  manageRoles: 0x10000000,
-  webhooks: 0x20000000,
-  emojis: 0x40000000
```

Comment:
Do you think it'd be a good idea to maybe replace these with their `<<`'s instead?
E.g
```js
export const bits = {
  createInvites: 1 << 0,
  kick: 1 << 1,
  // etc
```

Reply:
Yeah, that's probably a better way to implement it, I'll take care of that right now

Reply:
I just did some testing, as far as I can tell using left shift operators will cause unexpected behavior
The below code essentially mimics the `PermissionUtils.hasPerms()` method
```js
Number(BigInt(0x0100000000) & BigInt(0x0100000000)) // Output: 4294967296
Number(BigInt(1 << 32) & BigInt(1 << 32)) // Output: 1
```
This has adverse affects when comparing a single permission against multiple combined permissions